sonic-net/sonic-swss-common
# Add support of 'with' statement to ConfigDBConnector

#### Why I did it
Add support of 'with' statement to ConfigDBConnector

#### How I did it
Add __enter__ and __exit__ method to ConfigDBConnector

##### Work item tracking
- Microsoft ADO: 24222755

#### How to verify it
Pass all UT and E2E test cases.
Add new UT to check ConfigDBConnector support 'with' statement

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog
Add support of 'with' statement to ConfigDBConnector

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
